spec_opts_unix.go

```diff
@@ -111,22 +108,24 @@ func WithImageConfig(i Image) SpecOpts {
 					}
 					return err
```

Comment:
Why do we still return error here?

Reply:
Bad logic on the error handling, we should continue if WithUsername returns a nil error

Reply:
removing the `:` on line 109 should fix this.

Reply:
Will do.

Reply:
Done.